lib/src/shared/main/java/com/couchbase/lite/CouchbaseLiteException.java

```diff
@@ -108,18 +133,18 @@ public CouchbaseLiteException(@NonNull String message, @NonNull Throwable cause,
      * @param cause the internal exception
      */
     public CouchbaseLiteException(@NonNull CBLInternalException cause) {
```

Comment:
I'm just curious how this works in Java when the method is public but the argument is not.

Reply:
As you guessed, the method probably cannot be called.
It *could* be called, I suppose, if there were a public subtype of `CBLInternalException`
I think it is a mistake that *any* of these constructors are public.  Client code should *NOT* be creating new instances of `CouchbaseLiteException`.  The class and its accessors need to be public but I can't think of a single reason that the constructors should be.